app/src/main/res/layout/activity_confirm_request_leave.xml

```diff
@@ -30,309 +30,421 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 app:popupTheme="@style/AppTheme.PopupOverlay"
-                app:toolbar="@{viewModel.titleToolbar}"
-                />
+                >
+                <ImageView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:onClick="@{viewModel::onClickArrowBack}"
+                    android:src="@drawable/md_nav_back"
+                    tools:ignore="ContentDescription"
+                    />
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="@dimen/dp_35"
+                    android:layout_marginStart="@dimen/dp_35"
+                    android:text="@string/confirm_request_leave"
+                    android:textColor="@android:color/white"
+                    android:textSize="@dimen/sp_20"
+                    />
+            </android.support.v7.widget.Toolbar>
         </android.support.design.widget.AppBarLayout>
-
-        <ScrollView
+        <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:scrollbars="none"
             >
-            <LinearLayout
+            <ScrollView
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_margin="@dimen/dp_5"
-                android:orientation="vertical"
+                android:layout_height="match_parent"
+                android:layout_above="@+id/button_submit"
+                android:scrollbars="none"
                 >
-
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/dp_5"
                     android:orientation="vertical"
                     >
-                    <TableLayout
+
+                    <LinearLayout
```

Comment:
WARNING: [UselessParent] This `LinearLayout` layout or its `LinearLayout` parent is useless
 A layout with children that has no siblings, is not a scrollview or a root layout, and does not have a background, can be removed and have its children moved directly into the parent for a flatter and more efficient layout hierarchy.